bot/inputs/ft-disk/README.md

```diff
@@ -0,0 +1,4 @@
+Placeholder for testcases generated by fuzzer. This gets cleared after a task is finished.

```

Comment:
Aha! This is what I missed! It didn't occur to me that there might actually be a `fuzzer-testcases-disk` directory actually in the ClusterFuzz source tree. Thanks!